src/command_modules/azure-cli-hdinsight/azure/cli/command_modules/hdinsight/commands.py

```diff
@@ -6,17 +6,84 @@
 
 def load_command_table(self, _):
     from azure.cli.core.commands import CliCommandType
+    from ._client_factory import cf_hdinsight_applications
     from ._client_factory import cf_hdinsight_clusters
+    from ._client_factory import cf_hdinsight_extensions
+    from ._client_factory import cf_hdinsight_locations
+    from ._client_factory import cf_hdinsight_script_execution_history
+    from ._client_factory import cf_hdinsight_script_actions
 
     hdinsight_clusters_sdk = CliCommandType(
         operations_tmpl='azure.mgmt.hdinsight.operations.clusters_operations#ClustersOperations.{}',
         client_factory=cf_hdinsight_clusters
     )
 
+    hdinsight_script_actions_sdk = CliCommandType(
+        operations_tmpl='azure.mgmt.hdinsight.operations.script_actions_operations#ScriptActionsOperations.{}',
+        client_factory=cf_hdinsight_script_actions
+    )
+
+    hdinsight_applications_sdk = CliCommandType(
+        operations_tmpl='azure.mgmt.hdinsight.operations.applications_operations#ApplicationsOperations.{}',
+        client_factory=cf_hdinsight_applications
+    )
+
+    hdinsight_extensions_sdk = CliCommandType(
+        operations_tmpl='azure.mgmt.hdinsight.operations.extensions_operations#ExtensionsOperations.{}',
+        client_factory=cf_hdinsight_extensions
+    )
+
+    hdinsight_locations_sdk = CliCommandType(
+        operations_tmpl='azure.mgmt.hdinsight.operations.locations_operations#LocationsOperations.{}',
+        client_factory=cf_hdinsight_locations
+    )
+
+    hdinsight_script_execution_history_sdk = CliCommandType(
+        operations_tmpl='azure.mgmt.hdinsight.operations.script_execution_history_operations#'
+                        'ScriptExecutionHistoryOperations.{}',
+        client_factory=cf_hdinsight_script_execution_history
+    )
+
+    # cluster operations
     with self.command_group('hdinsight', hdinsight_clusters_sdk, client_factory=cf_hdinsight_clusters) as g:
         g.custom_command('create', 'create_cluster', supports_no_wait=True)
         g.command('resize', 'resize', supports_no_wait=True)
         g.show_command('show', 'get')
         g.custom_command('list', 'list_clusters')
         g.wait_command('wait')
         g.command('delete', 'delete', confirmation=True, supports_no_wait=True)
+
+    # usage operations
+    with self.command_group('hdinsight', hdinsight_locations_sdk, client_factory=cf_hdinsight_locations) as g:
+        g.command('usage', 'list_usages')
```

Comment:
change this to a verb, like `list-usage`

Reply:
Sounds good. Changed to `list-usage`